src/Compilers/CSharp/Portable/FlowAnalysis/DefiniteAssignment.VariableIdentifier.cs

```diff
@@ -80,7 +80,7 @@ public bool Equals(VariableIdentifier other)
                     return true;
                 }
 
-                return Symbol.OriginalDefinition.Equals(other.Symbol.OriginalDefinition, SymbolEqualityComparer.ConsiderEverything.CompareKind);
+                return Symbol.Equals(other.Symbol, SymbolEqualityComparer.IgnoreEverything.CompareKind);
```

Comment:
>SymbolEqualityComparer.IgnoreEverything.CompareKind [](start = 51, length = 51)

It looks like this is the only use of `SymbolEqualityComparer.IgnoreEverything`. That member can be removed and replaced here with `TypeCompareKind.AllIgnoreOptions`.